lib/extensions/descendant_loader.rb

```diff
@@ -0,0 +1,269 @@
+require 'ruby_parser'
```

Comment:
@matthewd can you do `require 'ruby22_parser'`?

```
  6  <+  | | | | | | ruby18_parser (0.201418) RES:2.89453125 VIRT:2.23046875
  6  <+  | | | | | | ruby19_parser (0.060308) RES:2.2578125 VIRT:2.4375
  6  <+  | | | | | | ruby20_parser (0.114312) RES:2.01171875 VIRT:2.45703125
  6  <+  | | | | | | ruby22_parser (0.059393) RES:1.30078125 VIRT:0.45703125
  5  <+  | | | | | ruby_parser (0.558660) RES:8.94140625 VIRT:8.0390625
```

Of the ~9 MB of RES memory and (1/2 second require on SSD) coming from the require, I think 7+ MB are for ruby 1.8, 1.9 and 2.0.  Are the latter "required"?